adapters/lora-vertx/src/test/java/org/eclipse/hono/adapter/lora/providers/FireflyProviderTest.java

```diff
@@ -58,6 +62,17 @@ public void extractTypeFromLoraUplinkMessage() {
         assertEquals(LoraMessageType.UPLINK, type);
     }
 
+    /**
+     * Verifies that the extracted mic is true
+     */
+    @Test
+    public void extractRssiFromLoraMessage() {
```

Comment:
sure that this should be called extractRssiFromLoraMessage and not extractMicFromLoraMessage ?

Reply:
you're right, fixed